diff --git a/node_modules/load-bmfont/browser.js b/node_modules/load-bmfont/browser.js
index 57e8f32..6465613 100644
--- a/node_modules/load-bmfont/browser.js
+++ b/node_modules/load-bmfont/browser.js
@@ -5,6 +5,7 @@ var parseXML = require('parse-bmfont-xml')
 var readBinary = require('parse-bmfont-binary')
 var isBinaryFormat = require('./lib/is-binary')
 var xtend = require('xtend')
+var { Buffer } = require('buffer')
 
 var xml2 = (function hasXML2() {
   return self.XMLHttpRequest && "withCredentials" in new XMLHttpRequest
diff --git a/node_modules/load-bmfont/index.js b/node_modules/load-bmfont/index.js
index f26f0b3..e2840c5 100644
--- a/node_modules/load-bmfont/index.js
+++ b/node_modules/load-bmfont/index.js
@@ -6,6 +6,7 @@ var parseASCII = require('parse-bmfont-ascii')
 var parseXML = require('parse-bmfont-xml')
 var readBinary = require('parse-bmfont-binary')
 var mime = require('mime')
+var { Buffer } = require('buffer')
 var noop = function() {}
 var isBinary = require('./lib/is-binary')
 
diff --git a/node_modules/load-bmfont/lib/is-binary.js b/node_modules/load-bmfont/lib/is-binary.js
index 5eb6790..0c0aec6 100644
--- a/node_modules/load-bmfont/lib/is-binary.js
+++ b/node_modules/load-bmfont/lib/is-binary.js
@@ -1,3 +1,4 @@
+var { Buffer } = require('buffer')
 var equal = require('buffer-equal')
 var HEADER = Buffer.from([66, 77, 70, 3])
 
